packages/ember-data/lib/serializers/json-serializer.js

```diff
@@ -682,7 +714,7 @@ export default Serializer.extend({
         }
       }
     });
-   ```
+    ```
```

Comment:
unfortunately the indentation in the comments is not always consistent and in this case it used the wrong indentation previously. it is now aligned properly with the rest of the comment.